batch/test/test_batch.py

```diff
@@ -310,13 +310,13 @@ def assert_batch_ids(expected: Set[int], q=None):
 
 
 def test_list_jobs(client: BatchClient):
-    b = client.create_batch()
-    j_success = b.create_job(DOCKER_ROOT_IMAGE, ['true'])
-    j_failure = b.create_job(DOCKER_ROOT_IMAGE, ['false'])
-    j_error = b.create_job(DOCKER_ROOT_IMAGE, ['sleep 5'], attributes={'tag': 'bar'})
-    j_running = b.create_job(DOCKER_ROOT_IMAGE, ['sleep', '1800'], attributes={'tag': 'foo'})
+    builder = client.create_batch()
```

Comment:
Ideally all these, which I assume are to fix type lints, would be in a separate PR not included in this PR for ease of review.